sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
@JimSuplizio This may be changed back to 1.1.0 depending on what requirements AppConfiguration has on Azure Core, but how are we planning to handle versioning as dependency versions begin to skew?